src/software/ai/hl/stp/tactic/stop_tactic.h

```diff
@@ -1,7 +1,41 @@
 #pragma once
 
-#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/action/move_action.h"  // TODO (#1888): remove this dependency
 #include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/ai/intent/stop_intent.h"
+
+struct StopFSM
+{
+    struct Stop
+    { /* state */
+    };
```

Comment:
Is this preferred over just using "Stop"_s (similar to "idle"_s)?

Reply:
Yes, because it prevents typos